src/include/miopen/conv/problem_description.hpp

```diff
@@ -248,6 +252,21 @@ struct ProblemDescription : ProblemDescriptionBase
     bool IsDirectionBackwardWrW() const { return direction == conv::Direction::BackwardWeights; }
     std::string GetDirectionStr() const;
 
+    const void* GetAlpha() const { return alpha; }
+    const void* GetBeta() const { return beta; }
+
+    bool IsDefaultAlphaBetaType() const
```

Comment:
I recall us identifying the following cases for `alpha/beta`: 
1. Default: both are null or alpha==1 and beta == 0
2. Alpha==x and beta == 0. This case will be handled by adding a `Scale` operation to CK instance
3. Alpha==x and beta==y. This case will be handled by adding a `BiLinear` operation to CK instance. 
4. Alpha==0 and beta==y should not occur but we should check for it as well. 

cc: @JehandadKhan .